src/components/SearchDialog.tsx

```diff
@@ -179,8 +195,67 @@ export default function SearchDialog(props: Props) {
               })}
             </RadioButton.Group>
           </SearchOptions>
+          <SearchOptions title="mode" icon="cog-outline">
```

Comment:
This (the public-facing UI part) is one of the bits I'm least sure about. I'm not convinced that "mode" with "offline" and "online" are the best names here (they feel maybe more technical than useful to the average person?) and I don't know that it conveys how to get back to something that just works if it turns out this is broken somehow. Other possibilities for the options include:
- "beta"/"stable" (though that's also tech-y)
- "experimental"/"reliable"
- "newer, faster"/"older, slower"

Similarly I'm not sure about the dialog text. Do we actually want to include a way for people to reach out about ways the new search is broken? What would that even be?

Reply:
I would leave out the contact part.

Part of me wants to switch this to a toggle button:

⚙ offline

off 🔲 ✅ on

Reply:
Hm, I'm not sure I've seen a toggle or switch quite like that, with the options on either side 🤔 I could certainly see a [switch](https://callstack.github.io/react-native-paper/docs/components/Switch/) labelled "enabled" or similar? Happy to give that shot if you'd like 😀